### DISCUSSION TITLE
rocm-opencl-sdk missing from AUR?

### DISCUSSION THREAD

**Discussion by Mushoz** [rocm-arch/rocm-arch#931] [Support] [answered]
The official command to install ROCM under Arch according to the README is the following command:

`paru -S rocm-hip-sdk rocm-opencl-sdk`

However, rocm-opencl-sdk is no longer available on the AUR and the page that once hosted it, gives a 404 page not found: https://aur.archlinux.org/packages/rocm-opencl-sdk

Was this package removed from the AUR by accident? Or should the README be updated?

**Comment by Mushoz** [comment]
On a similar note: "rocm-hip-libraries" (A dependency of "rocm-hip-sdk") is also gone from the AUR.

**Comment by tpkessler** [comment, marked as answer]
I pushed the new package structure to the AUR.
